BookPlayer/Player/PlayerViewController.swift

```diff
@@ -353,6 +355,17 @@ class PlayerViewController: UIViewController, UIGestureRecognizerDelegate {
         self.dismissPlayer()
         return true
     }
+
+    func updateAutolock() {
+        let autolockDisabled = UserDefaults.standard.bool(forKey: Constants.UserDefaults.autolockDisabled.rawValue)
+        let autolockDisabledOnlyWhenPowered = UserDefaults.standard.bool(forKey: Constants.UserDefaults.autolockDisabledOnlyWhenPowered.rawValue)
+        let charging = UIDevice.current.batteryState == .full || UIDevice.current.batteryState == .charging
```

Comment:
instead of going by `.full` and `.charging` we could check only for the `.unplugged` state

Reply:
I was slightly concerned about the [`.unknown` battery state](https://developer.apple.com/documentation/uikit/uidevice/batterystate/unknown) so chose to explicitly check for `.full`/`.charging`. I'm not sure how common it is for the battery state to be unknown...

But it definitely is cleaner to just check for `!= .unplugged`. 👍